### ISSUE TITLE
Error: Class 'Database\Factories\Entities\Units\UnitFactory' not found

### ISSUE THREAD

**Issue by nafiesl** [nafiesl/SimpleCrudGenerator#39]
## Descrption

Got error `Class 'Database\Factories\Entities\Units\UnitFactory' not found` when running unit tests after generating CRUD with a model in a different directory (other than `app/Models`).

## Package Version

- laravel/framework: v8.37.0
- luthfi/simple-crud-generator: 2.3.0

## How to Reproduce

1. `$ laravel new simple_crud_test`
2. `$ cd simple_crud_test`
2. Update `phpunit.xml` using sqlite in memory database
    ```
    <server name="DB_CONNECTION" value="sqlite"/>
    <server name="DB_DATABASE" value=":memory:"/>
    ```
3. `$ composer require laravel/ui`
4. `$ composer require luthfi/simple-crud-generator --dev`
5. `$ php artisan ui bootstrap --auth`
4. `$ php artisan make:crud Phone` (`Phone` model on the `app/Models` directory)
5. `$ vendor/bin/phpunit` (all tests are passed)
6. `$ php artisan make:crud Entities/Units/Unit` (`Unit` model on the `app/Entities/Units/Unit` directory)
7. `$ vendor/bin/phpunit --stop-on-failure`
8. Get `Error: Class 'Database\Factories\Entities\Units\UnitFactory' not found`

## Screenshot

![screen_2021-04-20_008](https://user-images.githubusercontent.com/8721551/115341823-d2b63b80-a1db-11eb-9e5e-c8fa17420105.jpg)

**Comment by nafiesl** [comment]
After done some research on the laravel/framework codebase, I found this line.
https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/Eloquent/Factories/Factory.php#L718-L720

Since Laravel 8.x, they are using class-based model factories, they need to automatically resolve the model factory class name based on the model class name.

If the file `Factory.php#L718-L720` they are checking if the class namespace is `App\Models`, the model factory class should be in `database/factories/ModelNameFactory`.

But when the Laravel model is stored outside the "app/Models", they will resolve the model factory class name based on the model's full namespace.

In my case, if my model path is `app/Entities/Units/Unit.php`, then they will look for model factory class on `database/factories/Entities/Units/UnitFactory.php`

I have created the fix on PR #40.
